assets/src/design-system/components/input/index.js

```diff
@@ -145,6 +174,7 @@ Input.propTypes = {
   disabled: PropTypes.bool,
   hasError: PropTypes.bool,
   hint: PropTypes.string,
+  Icon: PropTypes.elementType,
```

Comment:
I don't think the suffix is meant to only be an icon. It can also be a text. Going by our previous component, we just called this `suffix` and it could be whatever you passed to it.

Note that in [many panels](https://styled-components.com/docs/advanced#referring-to-other-components), the suffix is a letter rather than an actual icon.